src/python/nimbusml/tests/model_summary/test_model_summary.py

```diff
@@ -96,7 +96,23 @@ def test_model_summary(self):
                 [OneHotVectorizer() << categorical_columns, learner])
             train_stream = FileDataStream(train_file, schema=file_schema)
             pipeline.fit(train_stream, label_column)
-            pipeline.summary()
+            # If Pipeline does contain FactorizationMachineBinaryClassifier,
+            # check for TypeError thrown by pipeline as 
+            # FactorizationMachineBinaryClassifier does not support summary()
```

Comment:
This is not specific to Factorization Machine. Any other trainer, transformer needs to be handled.